include/oneapi/dpl/experimental/kt/esimd_radix_sort_onesweep.h

```diff
@@ -20,6 +20,79 @@
 
 namespace oneapi::dpl::experimental::esimd::impl
 {
+// float, 32bit
+template <typename T, bool __is_ascending, std::enable_if_t<std::is_same<T, float>::value && sizeof(T) == sizeof(::std::uint32_t), int> = 0>
+constexpr T
+__full_sort_identity()
+{
+    static_assert(std::is_same<T, float>::value, "");
+
+    //                                                                    sign bit
+    //                                                                      |      NaN bit
+    //                                                                      |        |
+    //                                                                      V        V
+    static_assert(::std::numeric_limits<T>::max()    == sycl::bit_cast<T>(0b01111111'01111111'11111111'11111111), "");
+    static_assert(::std::numeric_limits<T>::lowest() == sycl::bit_cast<T>(0b11111111'01111111'11111111'11111111), "");
+
+    if constexpr (__is_ascending)
+    {
+        //                       clear sign bit
+        //                         |      setup NaN bit
+        //                         |        |
+        //                         V        V
+        return sycl::bit_cast<T>(0b01111111'11111111'11111111'11111111);
```

Comment:
Let's use Hex representation. It is consistent with other functions doing bit-related operations.

Reply:
I believe we able to do this later. Just now this binary format more clear for me.

Reply:
We discussed it offline and agreed to leave it as is for easier debugging and return to it during productization.